erpnext/hr/utils.py

```diff
@@ -157,6 +157,16 @@ def get_employee_field_property(employee, fieldname):
 	else:
 		return False
 
+def get_company_set(company):
+	return frappe.db.sql_list("""
+		SELECT
+			name
+		FROM `tabCompany`
+		WHERE
+			parent_company=%(company)s
```

Comment:
It will only return immediate children companies. But if the company tree has multiple levels, then it will not return all companies. Use `lft` and `rgt` to fetch all children.